src/mapgen_functions.cpp

```diff
@@ -867,6 +867,17 @@ void mapgen_road( mapgendata &dat )
         }
     }
 
+    // place street lights
+    if( neighbor_sidewalks ) {
+        for( int x = 0; x < SEEX * 2 - 1; x++ ) {
+            for( int y = 0; y < SEEY * 2 - 1; y++ ) {
+                if( x % 5 == 0 && y % 5 == 0 && m->ter( point( x, y ) ) == t_sidewalk ) {
```

Comment:
```suggestion
        for( int x = 0; x < SEEX * 2 - 1; x += 5 ) {
            for( int y = 0; y < SEEY * 2 - 1; y += 5 ) {
                if( m->ter( point( x, y ) ) == t_sidewalk ) {
```
So much shorter.

Reply:
Although instead of 5, 15 is better. You need to leave space for benches and urns.

Reply:
Used 12-tile spacing as overmap is 24 tile long, and 15 would look... not so pretty.